From 1e971c8d3150f8057c12a99361a3ac6ec1b8ae37 Mon Sep 17 00:00:00 2001
From: Alexander Kuleshov <kuleshovmail@gmail.com>
Date: Sat, 27 Jun 2015 19:37:22 +0600
Subject: [PATCH 4/4 v13] x86/earlyprintk: add some early_printk for tests

This patch is only for test of the full patch series. It provides
a couple calls of the early_printk function.

[Only for testing]

Signed-off-by: Alexander Kuleshov <kuleshovmail@gmail.com>
---
 arch/x86/kernel/setup.c | 1 +
 init/main.c             | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/arch/x86/kernel/setup.c b/arch/x86/kernel/setup.c
index fc381ec..488b1ba 100644
--- a/arch/x86/kernel/setup.c
+++ b/arch/x86/kernel/setup.c
@@ -878,6 +878,7 @@ void __init setup_builtin_cmdline(void)
 
 void __init setup_arch(char **cmdline_p)
 {
+	early_printk("Beginning of the x86 specific setup\n");
 	memblock_reserve(__pa_symbol(_text),
 			 (unsigned long)__bss_stop - (unsigned long)_text);
 
diff --git a/init/main.c b/init/main.c
index c599aea..deeb2a2 100644
--- a/init/main.c
+++ b/init/main.c
@@ -499,6 +499,8 @@ asmlinkage __visible void __init start_kernel(void)
 	char *command_line;
 	char *after_dashes;
 
+	early_printk("start_kernel from init/main.c started to work\n");
+
 	/*
 	 * Need to run as early as possible, to initialize the
 	 * lockdep hash:
--
2.4.4.410.gc71d752

